Assets/MixedRealityToolkit.Examples/Demos/Solvers/Profiles/SurfaceMagnetismExampleSpatialAwarenessSystemProfile.asset

```diff
@@ -0,0 +1,23 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 19f279aded72cb741b4de89a54359dd4, type: 3}
```

Comment:
Will you be removing this asset, per Troy's comment?

Reply:
Yes. And adding toggle button for spatial mesh visual.